### PR TITLE
PEP8 changes

### PR DESCRIPTION
**Pull request by kain88-de**
Changes made in this Pull Request:
 -  just clean up most complains of my PEP8 checker (flake8)

I'm sure this fixed some bugs due to missing imports. I haven't written tests for that code yet though.

PR Checklist
------------
 - [ ] Tests?

### CODE REVIEW

File: package/MDAnalysis/core/groups.py [hunk 1]
```diff
@@ -907,7 +913,7 @@ def forces(self):
     def forces(self, values):
         ts = self._u.trajectory.ts
         try:
-            ts.forces[self._ix, :] = forces
+            ts.forces[self._ix, :] = values
```
Comment (richardjgowers) [inline package/MDAnalysis/core/groups.py]: This should have tripped a test....

File: package/MDAnalysis/core/groups.py [hunk 2]
```diff
@@ -2,7 +2,8 @@
 # vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
 #
 # MDAnalysis --- http://www.MDAnalysis.org
-# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
```
Comment (orbeckst) [inline package/MDAnalysis/core/groups.py]: If you change this here, will @tylerjereddy 's #1002 still work?

File: package/MDAnalysis/core/groups.py [hunk 3]
```diff
@@ -20,11 +21,12 @@
 import numpy as np
 import functools
 import itertools
+import os
```
Comment (orbeckst) [inline package/MDAnalysis/core/groups.py]: Really, `os` was used but not imported and did not fail a test?

File: package/MDAnalysis/core/groups.py (hunk 1 above)
Comment (kain88-de) [inline package/MDAnalysis/core/groups.py]: one would think so. I'm currently writing tests for the write function. If you find more spots where tests are needed to be triggered by this change please mark them

File: package/MDAnalysis/core/groups.py (hunk 3 above)
Comment (kain88-de) [inline package/MDAnalysis/core/groups.py]: No the same goes for transformations. I'm writing tests now that are affected by the changes. But this just shows how important a good style checker is.

File: package/MDAnalysis/core/groups.py (hunk 2 above)
Comment (kain88-de) [inline package/MDAnalysis/core/groups.py]: No idea. I can remove this one change though. But we have this mixed format in almost all the files already. Everytime I changed the formatting in some random file.

**Comment by orbeckst** [comment]
Ok, then Tyler will have to deal with it.

> On 7 Oct, 2016, at 12:20, Max Linke <notifications@github.com> wrote:
> 
> No idea. I can remove this one change though. But we have this mixed format in almost all the files already. Everytime I changed the formatting in some random file.

--
Oliver Beckstein * orbeckst@gmx.net
skype: orbeckst  * orbeckst@gmail.com

**Comment by kain88-de** [comment]
@richardjgowers I'd appreciate some help with the failing travis tests. Not sure if I accidentally broke something with this.

**Comment by richardjgowers** [comment]
Those 4 failures are OK, they got added with some recent new tests Kyle added

File: package/MDAnalysis/core/groups.py (hunk 3 above)
Comment (kain88-de) [inline package/MDAnalysis/core/groups.py]: This is tested now.

File: package/MDAnalysis/core/groups.py (hunk 1 above)
Comment (kain88-de) [inline package/MDAnalysis/core/groups.py]: where would old tests for this be?

**Comment by kain88-de** [comment]
OK after looking into develop the transformations tests have never existed. So I will take care of them with #1010. @richardjgowers if you want to take care of the force setter tests I would consider this done.

**Comment by richardjgowers** [comment]
Force setter tests I think are in something called
test_velocities_forces.py

**Comment by kain88-de** [comment]
I've also added tests for the forces now.

**Pull request merged**